site/sass/components/_card-subnav.scss

```diff
@@ -5,43 +5,79 @@
   $inline-padding: pxRem(28px);
 
   align-items: center;
+  background-color: var(--white);
+  border-radius: 0.625rem;
   display: flex;
   flex-direction: column;
   margin: 0 auto;
-  max-width: 40ch;
+  max-width: 52ch;
+  padding-bottom: 2rem;
+  padding-top: 1.625rem;
 
   &__header {
     align-items: center;
     display: flex;
-    flex-direction: column;
+    justify-content: space-between;
+    margin-bottom: 0.875rem;
     padding: 0 $inline-padding;
+    width: 100%;
+
+    @supports (margin-bottom: clamp(1px, 1vw, 2px)) {
+      margin-bottom: clamp(0.875rem, 2.66vw, 1.5rem);
+    }
   }
 
   &__title {
-    margin-bottom: pxRem(8px);
     text-align: center;
   }
 
+  &__title,
   &__body {
-    margin-bottom: pxRem(40px);
+    color: var(--grey-850);
+  }
+
+  &__body,
+  &__sections {
     padding: 0 $inline-padding;
-    text-align: center;
+  }
+
+  &__body {
+    margin-bottom: 1.25rem;
   }
 
   &__icon-wrapper {
     align-items: center;
     background-color: var(--blue-600);
     border-radius: 50%;
     display: flex;
-    height: pxRem(96px);
+    flex-shrink: 0;
+    height: 3rem;
     justify-content: center;
-    margin-bottom: pxRem(24px);
-    width: pxRem(96px);
+    width: 3rem;
+
+    @supports (width: clamp(1px, 1vw, 2px)) {
+      height: clamp(2.5rem, 7.4vw, 3rem);
+      width: clamp(2.5rem, 7.4vw, 3rem);
+    }
+
+    &--primary-blue {
+      background-color: var(--primary-blue);
+    }
+
+    &--primary-yellow {
+      background-color: var(--primary-yellow);
+    }
+
+    &--primary-green {
+      background-color: var(--primary-green);
+    }
```

Comment:
Same comments from the component itself; let's not couple these directly to the CSS var name